server/services/participant-status.js

```diff
@@ -102,7 +102,7 @@ const setParticipantStatus = async (
       : await tx[collections.PARTICIPANTS_STATUS].findOne({
           participant_id: participantId,
           current: true,
-          or: [{ employer_id: employerId }, { 'data.site IN': user.sites || [] }],
+          employer_id: employerId,
```

Comment:
Reverting this to the way it was before. If we properly pass in the status id, we shouldn't need to check this clause for user sites. I'm worried I introduced some badness.